server.js

```diff
@@ -52,6 +52,25 @@ var _ = require('underscore'),
 // requires content-type application/json header
 var jsonParser = bodyParser.json({limit: '32mb'});
 
+const handleJsonRequest = (method, path, callback) => {
+  app[method](path, jsonParser, (req, res) => {
+    const contentType = req.headers['content-type'];
+    if (!contentType || contentType !== 'application/json') {
+      return serverUtils.error({
+        code: 400,
+        message: 'Content-Type must be application/json'
+      }, req, res);
+    } else {
+      callback(req, res);
+    }
+  });
+};
+['delete', 'post', 'put'].forEach(method => {
+  app[method + 'Json'] = (path, callback) => {
```

Comment:
This makes it _really_ hard to follow when debugging because I won't be able to search for the "postJson" string. Can you change this to...

```js
['deleteJson', 'postJson', 'putJson'].forEach(method => {
  app[method] = (path, callback) => {
```

It's only costs a few characters but is much easier to follow IMO.